### DISCUSSION TITLE
Decorators with storybook html?

### DISCUSSION THREAD

**Discussion by usma1000** [storybookjs/storybook#13953] [Help]
I'm trying to use decorators in CSF format. I tried this example in the documentation, but since it's written for React, it didn't work for me.

```
export default {
  title: 'My Component',
  decorators: [(Story) => <div style={{ margin: '3em' }}><Story/></div>]
}
```

Is it currently possible to use decorators on a component or story level for html?

**Comment by shilman** [comment]
See https://github.com/storybookjs/storybook/issues/4997#issuecomment-447292895

cc @jonniebigodes
